slirp4netns.1.md

```diff
@@ -59,6 +59,11 @@ specify network namespace type ([path|pid], default=pid)
 **--userns-path=PATH** (since v0.4.0)
 specify user namespace path
 
+**--create-sandbox** (since v0.5.0)
```

Comment:
I think we can have this in v0.4.0 as an experimental feature

Reply:
changed